Find a functionally equivalent code:android.app.SearchDialog.createContentView:COMMENT
Method Modifier: private     
Comment:/**
 * We recreate the dialog view each time it becomes visible so as to limit
 * the scope of any problems with the contained resources.
 */

@@ -22,28 +22,28 @@
     mSearchAutoComplete = (AutoCompleteTextView) mSearchView.findViewById(com.android.internal.R.id.search_src_text);
     mAppIcon = (ImageView) findViewById(com.android.internal.R.id.search_app_icon);
     mSearchPlate = mSearchView.findViewById(com.android.internal.R.id.search_plate);
-    mWorkingSpinner = getContext().getResources().getDrawable(com.android.internal.R.drawable.search_spinner);
+    mWorkingSpinner = getContext().getDrawable(com.android.internal.R.drawable.search_spinner);
     // TODO: Restore the spinner for slow suggestion lookups
     // mSearchAutoComplete.setCompoundDrawablesWithIntrinsicBounds(
     // null, null, mWorkingSpinner, null);

